Fix helm-rails tests

jferris opened the pull request:
- Ignore tmp/vendor directories from tests
- Fix selector in HPA tests

jferris committed:
Ignore tmp/vendor directories from tests

jferris committed:
Fix selector in HPA tests

OlamideOl1 commented:
We may need to bump the chart version from `0.1.6` to  maybe `0.1.7` to fix the lint error.

jferris committed:
Bump chart version

jferris committed:
Remove trailing space

jferris committed:
Add chart maintainers

jferris committed:
Specify target branch during helm test install

jferris committed:
Update version of chart-testing-action

jferris committed:
Install Istio CRDs during tests

jferris committed:
Fix namespace and serviceaccount creation

- Don't try to create the "default" namespace
- Create service account if specified even without secretproviderclass

jferris committed:
Deep copy values to avoid recursive structure

jferris committed:
Add tool versions for kind, kubectl, helm
